pkg/apis/build/v1beta1/buildrun_conversion.go

```diff
@@ -232,9 +240,17 @@ func (src *BuildRun) ConvertFrom(ctx context.Context, obj *unstructured.Unstruct
 		}
 	}
 
+	var output *Output
+	if alphaBuildRun.Status.Output != nil {
```

Comment:
I'm not sure if this is needed, because alpha api does not have the `.Vulnerabilities` field, so just converting the whole output might be ok. @SaschaSchwarze0 any opinions?

Reply:
The Output types are different for alpha and beta now, so it was not allowing to convert the whole output.